### PR TITLE
Improve Spanish Translation

### LINKED ISSUE #643: Improve Spanish translation of MASVS
I have some tips to improve a bit the Spanish translation, just to improve comprehesion, sometimes I feel the paragraph is a literal translation from English, some little changes will improve comphesion.

Others changes could be apply to the global section too, for example when you mention a chapter, it will be great if mention the number too.
For example: 
'On the chapter Resilience on apart "Device Binding"' 
VS
'On the chapter Resilence (V8) on apart "Device Binding" (8.10)

Thanks.

### PR DESCRIPTION
**Pull request by antoniojturel**
Thank you for submitting a Pull Request to the Mobile Application Security Verification Standard. Please make sure that:

- [ ] Your PR is linked to an issue.
- [ ] Your PR follows the [contribution guidelines](https://github.com/OWASP/owasp-masvs/blob/master/CONTRIBUTING.md "Contribution guidelines")
- [ ] All links are valid and use HTTPS and have been formatted as [TEXT](URL “NAME”)
- [ ] In case of a translation: have you made sure that the translation from EN is as close to the EN document as possible?

This PR closes #643.

If your pull-request has a focus of rewriting a requirement and/or adding a new one, please make sure you have a backing issue in which your change has been discussed and agreed. If there is none, then your PR will be closed without further review.

### CODE REVIEW

File: Document-es/0x01-Foreword.md [hunk 1]
```diff
@@ -1,24 +1,24 @@
...
+Además, la aparición del malware y las herramientas de administración remota han concienciado en que los propios sistemas operativos móviles tienen vulnerabilidades explotables, por lo que las estrategias de aislamiento se utilizan cada vez más para proporcionar protección adicional a los datos confidenciales y evitar la manipulación en el lado del cliente. Aquí es donde las cosas se complican. Las características de seguridad por hardware y las soluciones de aislamiento a nivel de sistema operativo, como _Android for Work_ y Samsung Knox, existen, pero no siempre están disponibles en diferentes dispositivos. A modo de _curita o tirita_, es posible implementar medidas de protección basadas en software - pero desafortunadamente, no hay estándares o procesos de prueba para verificar este tipo de protecciones.
```
Comment (cpholguera) [inline Document-es/0x01-Foreword.md:13]:
```suggestion
Además, la aparición del malware y las herramientas de administración remota han conseguido promover una mayor conciencia pública de que los propios sistemas operativos móviles tienen vulnerabilidades explotables, por lo que las estrategias de aislamiento se utilizan cada vez más para proporcionar protección adicional a los datos confidenciales y evitar la manipulación en el lado del cliente. Aquí es donde las cosas se complican. Las características de seguridad por hardware y las soluciones de aislamiento a nivel de sistema operativo, como _Android for Work_ y Samsung Knox, existen, pero no siempre están disponibles en diferentes dispositivos. Es posible implementar medidas de protección basadas en software a modo de parches - pero desafortunadamente, no hay estándares o procesos de prueba para verificar este tipo de protecciones.
```

File: Document-es/0x01-Foreword.md [hunk 2]
```diff
@@ -1,24 +1,24 @@
...
+Como resultado, los informes de pruebas de seguridad de aplicaciones móviles están por todas partes: Por ejemplo, algunos reportan una falta de ofuscación o detección de _root_ en una aplicación Android como "fallo de seguridad". Por otra parte, las medidas como el cifrado de _strings_, la detección de depuradores o la ofuscación del flujo de control no se consideran obligatorias. Sin embargo, esta forma binaria de ver las cosas no tiene sentido porque la resistencia no es una proposición binaria sino que depende de las amenazas particulares en el dispositivo contra las que se quiere defender. Las protecciones de software no son inútiles, pero en última instancia pueden ser eludidas, por lo que nunca deben utilizarse como sustituto de los controles de seguridad.
```
Comment (cpholguera) [inline Document-es/0x01-Foreword.md:15]:
```suggestion
Como resultado, los informes de pruebas de seguridad de aplicaciones móviles están por todas partes: por ejemplo, algunos reportan una falta de ofuscación o detección de _root_ en una aplicación Android como "fallo de seguridad". Por otra parte, las medidas como el cifrado de _strings_, la detección de depuradores o la ofuscación del flujo de control no se consideran obligatorias. Sin embargo, esta forma binaria de ver las cosas no tiene sentido porque la resistencia no es una proposición binaria sino que depende de las amenazas particulares en el dispositivo contra las que se quiere defender. Las protecciones de software no son inútiles, pero en última instancia pueden ser eludidas, por lo que nunca deben utilizarse como sustituto de los controles de seguridad.
```

File: Document-es/0x01-Foreword.md [hunk 3]
```diff
@@ -1,24 +1,24 @@
...
 
-El objetivo general del MASVS es ofrecer una línea de base para la seguridad de las aplicaciones móviles (MASVS-L1), mientras que también permite la inclusión de medidas de defensa en profundidad (MASVS-L2) y protecciones contra las amenazas del lado del cliente (MASVS-R). El MASVS está pensado para lograr lo siguiente:
+El objetivo general del MASVS es ofrecer tanto una línea de base para la seguridad de las aplicaciones móviles (MASVS-L1) como también la inclusión de medidas de defensa en profundidad (MASVS-L2) y protecciones contra las amenazas del lado del cliente (MASVS-R). El MASVS está pensado para lograr lo siguiente:
```
Comment (cpholguera) [inline Document-es/0x01-Foreword.md:17]:
```suggestion
El objetivo general del MASVS es ofrecer tanto una línea base de referencia para la seguridad de las aplicaciones móviles (MASVS-L1) como también la inclusión de medidas de defensa en profundidad (MASVS-L2) y protecciones contra las amenazas del lado del cliente (MASVS-R). El MASVS está pensado para lograr lo siguiente:
```

File: Document-es/0x03-Using_the_MASVS.md [hunk 4]
```diff
@@ -50,7 +50,7 @@ En resumen, los siguientes tipos de verificación están disponibles:
 - MASVS-L2
 - MASVS-L2+R
 
-Las diferentes combinaciones reflejan diferentes grados de seguridad y resistencia. El objetivo es permitir la flexibilidad: por ejemplo, un juego móvil puede no requerir controles de seguridad MASVS-L2, como la autenticación de doble factor por razones de usabilidad, pero seguramente deba intentar prevenir la manipulación del código por razones de negocio.
+Las diferentes combinaciones reflejan diferentes grados de seguridad y resistencia. El objetivo es permitir la flexibilidad: Por ejemplo, un juego móvil puede no requerir controles de seguridad MASVS-L2, como la autenticación de doble factor por razones de usabilidad, pero seguramente deba intentar prevenir la manipulación del código por razones de negocio.
```
Comment (cpholguera) [inline Document-es/0x03-Using_the_MASVS.md:53]:
```suggestion
Las diferentes combinaciones reflejan diferentes grados de seguridad y resistencia. El objetivo es permitir la flexibilidad: por ejemplo, un juego móvil puede no requerir controles de seguridad MASVS-L2, como la autenticación de doble factor por razones de usabilidad, pero seguramente deba intentar prevenir la manipulación del código por razones de negocio.
```

File: Document-es/0x03-Using_the_MASVS.md [hunk 5]
```diff
@@ -60,23 +60,23 @@ La implementación de los requisitos del nivel MASVS-L2 aumenta la seguridad, pe
...
+- Industria de la Salud: Aplicaciones móviles que almacenan información personal identificable que puede ser utilizada para el robo de identidad, pagos fraudulentos, o una variedad de esquemas de fraude. Para el sector de la salud en los Estados Unidos, las consideraciones de cumplimiento incluyen la Ley de Portabilidad y Responsabilidad del Seguro Médico (HIPAA, por sus siglas en inglés), Privacidad, Seguridad, Reglas de Notificación de Violación (Breach Notification Rules) y Reglas de Seguridad del Paciente (Patient Safety Rule).
 
 - Sector Financiero: Aplicaciones que permiten el acceso a información altamente sensible como números de tarjetas de crédito, información personal o que permiten al usuario mover fondos. Estas aplicaciones deben tener controles de seguridad adicionales para prevenir el fraude. Las aplicaciones financieras necesitan asegurar el cumplimiento de las normas de seguridad de datos de la industria de tarjetas de pago (PCI DSS), Gramm Leech Bliley Act y Sarbanes-Oxley Act (SOX).
```
Comment (cpholguera) [inline Document-es/0x03-Using_the_MASVS.md:69]:
```suggestion
- Industria de la Salud: aplicaciones móviles que almacenan información personal identificable que puede ser utilizada para el robo de identidad, pagos fraudulentos, o una variedad de esquemas de fraude. Para el sector de la salud en los Estados Unidos, las consideraciones de cumplimiento incluyen la Ley de Portabilidad y Responsabilidad del Seguro Médico (HIPAA, por sus siglas en inglés), Privacidad, Seguridad, Reglas de Notificación de Violación (Breach Notification Rules) y Reglas de Seguridad del Paciente (Patient Safety Rule).

- Sector Financiero: aplicaciones que permiten el acceso a información altamente sensible como números de tarjetas de crédito, información personal o que permiten al usuario mover fondos. Estas aplicaciones deben tener controles de seguridad adicionales para prevenir el fraude. Las aplicaciones financieras necesitan asegurar el cumplimiento de las normas de seguridad de datos de la industria de tarjetas de pago (PCI DSS), Gramm Leech Bliley Act y Sarbanes-Oxley Act (SOX).
```

File: Document-es/0x03-Using_the_MASVS.md [hunk 6]
```diff
@@ -60,23 +60,23 @@ La implementación de los requisitos del nivel MASVS-L2 aumenta la seguridad, pe
...
 
-- Industria de los juegos: juegos con una necesidad esencial de evitar la posibilidad de _modding_ y el engaño, como los juegos competitivos _online_. Hacer trampa es un tema importante en los juegos _online_, ya que una gran cantidad de tramposos conduce a un descontento de la base de jugadores y, en última instancia, puede causar que un juego fracase. MASVS-R proporciona controles básicos contra la manipulación para ayudar a incrementar el esfuerzo que los tramposos deberían realizar.
+- Industria de los videojuegos: Aquellos con una necesidad esencial de evitar la posibilidad de _modding_ y el engaño, como los videojuegos competitivos _online_. Hacer trampa es un tema importante en los juegos _online_, ya que una gran cantidad de tramposos conduce a un descontento de la base de jugadores y, en última instancia, puede causar que un juego fracase. MASVS-R proporciona controles básicos contra la manipulación para ayudar a incrementar el esfuerzo que los tramposos deberían realizar.
```
Comment (cpholguera) [inline Document-es/0x03-Using_the_MASVS.md:75]:
```suggestion
- Industria de los videojuegos: aplicaciones con una necesidad esencial de evitar la posibilidad de _modding_ y el engaño, como los videojuegos competitivos _online_. Hacer trampa es un tema importante en los juegos _online_, ya que una gran cantidad de tramposos conduce a un descontento de la base de jugadores y, en última instancia, puede causar que un juego fracase. MASVS-R proporciona controles básicos contra la manipulación para ayudar a incrementar el esfuerzo que los tramposos deberían realizar.
```

File: Document-es/0x03-Using_the_MASVS.md [hunk 7]
```diff
@@ -60,23 +60,23 @@ La implementación de los requisitos del nivel MASVS-L2 aumenta la seguridad, pe
...
 
 ###### MASVS L2+R
 
-- Industria Financiera: aplicaciones de banca móvil que permiten al usuario mover fondos, donde las técnicas de inyección de código e instrumentación en dispositivos comprometidos suponen un riesgo. En este caso, los controles MASVS-R se pueden utilizar para dificultar la manipulación de código, dificultando el trabajo de potenciales autores de _malware_.
+- Industria Financiera: Aplicaciones de banca móvil que permiten al usuario mover fondos, donde las técnicas de inyección de código e instrumentación en dispositivos comprometidos suponen un riesgo. En este caso, los controles MASVS-R se pueden utilizar para dificultar la manipulación de código, dificultando el trabajo de potenciales autores de _malware_.
```
Comment (cpholguera) [inline Document-es/0x03-Using_the_MASVS.md:79]:
```suggestion
- Industria Financiera: aplicaciones de banca móvil que permiten al usuario mover fondos, donde las técnicas de inyección de código e instrumentación en dispositivos comprometidos suponen un riesgo. En este caso, los controles MASVS-R se pueden utilizar para dificultar la manipulación de código, dificultando el trabajo de potenciales autores de _malware_.
```

File: Document-es/0x04-Assessment_and_Certification.md [hunk 8]
```diff
@@ -23,15 +23,15 @@ Los informes creados por organismos de certificación deberían incluir los sigu
...
 
 ### El Papel de las Herramientas de Pruebas de Seguridad Automatizadas
 
-Se recomienda el uso de escáneres de código fuente y herramientas de auditoría de tipo _black-box_ para aumentar la eficiencia siempre que sea posible. Sin embargo, no es posible completar la verificación MASVS utilizando únicamente herramientas automatizadas: cada aplicación móvil es diferente. Por tanto, la comprensión de la arquitectura general, la lógica de negocio y los problemas específicos de las tecnologías y plataformas que se utilizan es un requerimiento obligatorio para verificar la seguridad de la aplicación.
+Se recomienda el uso de escáneres de código fuente y herramientas de auditoría de tipo _black-box_ para aumentar la eficiencia siempre que sea posible. Sin embargo, no es posible completar la verificación MASVS utilizando únicamente herramientas automatizadas: Cada aplicación móvil es diferente. Por tanto, la comprensión de la arquitectura general, la lógica de negocio y los problemas específicos de las tecnologías y plataformas que se utilizan es un requerimiento obligatorio para verificar la seguridad de la aplicación.
```
Comment (cpholguera) [inline Document-es/0x04-Assessment_and_Certification.md:34]:
```suggestion
Se recomienda el uso de escáneres de código fuente y herramientas de auditoría de tipo _black-box_ para aumentar la eficiencia siempre que sea posible. Sin embargo, no es posible completar la verificación MASVS utilizando únicamente herramientas automatizadas: cada aplicación móvil es diferente. Por tanto, la comprensión de la arquitectura general, la lógica de negocio y los problemas específicos de las tecnologías y plataformas que se utilizan es un requerimiento obligatorio para verificar la seguridad de la aplicación.
```

File: Document-es/0x07-V2-Data_Storage_and_Privacy_requirements.md [hunk 9]
```diff
@@ -1,40 +1,40 @@
...
+Los datos sensibles en el contexto del MASVS se refieren tanto a las credenciales de usuario como a cualquier otro dato que se considere sensible en el contexto particular de los siguientes ejemplos:
 
-- Información de identificación personal que puede ser usada para el robo de identidad: números de seguro social, números de tarjetas de crédito, números de cuentas bancarias, información médica;
-- Datos altamente confidenciales que, en caso de que se comprometieran, ocasionarían daños a la reputación y/o costes financieros: información contractual, información cubierta por acuerdos de confidencialidad, información de gestión;
+- Información de identificación personal que puede ser usada para el robo de identidad: Información de la Seguridad Social, números de tarjetas de crédito, números de cuentas bancarias, información médica.
+- Datos altamente confidenciales que, en caso de que ser comprometidos, ocasionarían daños a la reputación y/o costes financieros: Información contractual, información cubierta por acuerdos de confidencialidad, información de gestión.
```
Comment (cpholguera) [inline Document-es/0x07-V2-Data_Storage_and_Privacy_requirements.md:14]:
```suggestion
- Datos altamente confidenciales que, en caso de que ser comprometidos, ocasionarían daños a la reputación y/o costes financieros: información contractual, información cubierta por acuerdos de confidencialidad, información de gestión.
```

File: Document-es/0x09-V4-Authentication_and_Session_Management_Requirements.md [hunk 10]
```diff
@@ -1,24 +1,24 @@
...
-| **4.5** | MSTG-AUTH-5 | Existe una política de contraseñas y es aplicada en el servidor. | x | x |
-| **4.6** | MSTG-AUTH-6 | El servidor implementa mecanismos, cuando credenciales de autenticación son ingresadas una cantidad excesiva de veces. | x | x |
-| **4.7** | MSTG-AUTH-7 | Las sesiones y los tokens de acceso expiran luego de un tiempo predefinido de inactividad. | x  | x |
-| **4.8** | MSTG-AUTH-8 | La autenticación biométrica, si la hay, no está asociada a eventos (p. ej. usando una API que simplemente retorna "true" o "false"), sino basada en el desbloqueo del keychain/keystore (almacenamiento seguro). |   | x |
-| **4.9** | MSTG-AUTH-9 | El sistema remoto implementa un mecanismo de segundo factor de autenticación (2FA) y lo impone consistentemente. |   | x |
+| **4.5** | MSTG-AUTH-5 | Debe existir una política de contraseñas y debe ser aplicada en el servidor. | x | x |
+| **4.6** | MSTG-AUTH-6 | El servidor debe implementar mecanismos cuando se excede un determinado número de intentos de inicio de sesión. Es decir, cuando se introducen erroneamente las credenciales de autenticación. | x | x |
```
Comment (cpholguera) [inline Document-es/0x09-V4-Authentication_and_Session_Management_Requirements.md:16]: No podemos cambiarlo tanto respecto al original en inglés. Qué te parece esto?

```suggestion
| **4.6** | MSTG-AUTH-6 | El servidor debe implementar mecanismos que eviten excesivos de intentos de inicio de sesión. | x | x |
```

File: Document-es/0x09-V4-Authentication_and_Session_Management_Requirements.md [hunk 11]
```diff
@@ -1,24 +1,24 @@
...
+| **4.7** | MSTG-AUTH-7 | Las sesiones y los tokens de acceso deben expirar pasado un tiempo predefinido de inactividad. | x  | x |
+| **4.8** | MSTG-AUTH-8 | La autenticación biométrica, si la hay, no debe estar asociada a eventos (p. ej. usando una API que simplemente retorna "true" o "false"), sino basada en el desbloqueo del keychain/keystore (almacenamiento seguro). |   | x |
+| **4.9** | MSTG-AUTH-9 | En el dispositivo final debe existir un mecanismo de segundo factor de autenticación (2FA) y éste debe ser consistente. |   | x |
```
Comment (cpholguera) [inline Document-es/0x09-V4-Authentication_and_Session_Management_Requirements.md:19]:
```suggestion
| **4.9** | MSTG-AUTH-9 | En el servidor debe existir un mecanismo de segundo factor de autenticación (2FA) y éste debe ser aplicado de forma consistente. |   | x |
```

File: Document-es/0x10-V5-Network_communication_requirements.md [hunk 12]
```diff
@@ -1,19 +1,19 @@
...
-| **5.5** | MSTG-NETWORK-5 | La aplicación no depende de un único canal de comunicaciones inseguro (email o SMS) para operaciones críticas como registro de usuarios o recuperación de cuentas. |  | x |
-| **5.6** | MSTG-NETWORK-6 | La aplicación sólo depende de bibliotecas de conectividad y seguridad actualizadas. |  | x |
+| **5.1** | MSTG-NETWORK-1 | La información se debe enviar cifrada utilizando TLS. El canal seguro es usado consistentemente en la aplicación. | x | x |
+| **5.2** | MSTG-NETWORK-2 | Las configuraciones del protocolo TLS deben seguir las buenas prácticas de la industria, o deben hacerlo lo mejor posible en caso de que el sistema operativo del dispositivo no soporte los estándares recomendados. | x | x |
+| **5.3** | MSTG-NETWORK-3 | La aplicación debe verificar el certificado X.509 del sistema remoto al establecer el canal seguro y sólo se deben aceptar certificados firmados por una entidad certificadora (CA) de confianza. | x | x |
```
Comment (cpholguera) [inline Document-es/0x10-V5-Network_communication_requirements.md:13]:
```suggestion
| **5.3** | MSTG-NETWORK-3 | La aplicación debe verificar el certificado X.509 del sistema remoto al establecer el canal seguro y sólo se deben aceptar certificados firmados por una autoridad certificadora (CA) de confianza. | x | x |
```

File: Document-es/0x10-V5-Network_communication_requirements.md [hunk 13]
```diff
@@ -1,19 +1,19 @@
...
+| **5.4** | MSTG-NETWORK-4 | La aplicación debe utilizar su propio almacén de certificados o realiza _pinning_ del certificado o la clave pública del servidor. Bajo ningún concepto establecerá conexiones con servidores que ofrecen otros certificados o claves, incluso si están firmados por una entidad certificadora (CA) de confianza. |   | x |
```
Comment (cpholguera) [inline Document-es/0x10-V5-Network_communication_requirements.md:14]:
```suggestion
| **5.4** | MSTG-NETWORK-4 | La aplicación debe utilizar su propio almacén de certificados o realiza _pinning_ del certificado o la clave pública del servidor. Bajo ningún concepto establecerá conexiones con servidores que ofrecen otros certificados o claves, incluso si están firmados por una autoridad certificadora (CA) de confianza. |   | x |
```

File: Document-es/0x11-V6-Interaction_with_the_environment.md [hunk 14]
```diff
@@ -1,24 +1,24 @@
...
-| **6.9** | MSTG-PLATFORM-9 | La aplicación se protege contra ataques de tipo screen overlay. (sólo Android) |  | x |
-| **6.10** | MSTG-PLATFORM-10 | La caché, el almacenamiento y los recursos cargados (JavaScript, etc.) de las WebViews deben de borrarse antes de destruir la WebView. |  | x |
-| **6.11** | MSTG-PLATFORM-11 | Verificar que la aplicación impide el uso de teclados de terceros siempre que se introduzca información sensible. (sólo iOS) |  | x |
+| **6.3** | MSTG-PLATFORM-3 | La aplicación no debe exponer ninguna funcionalidad sensible a través de esquemas de URL salvo que dichos mecanismos estén debidamente protegidos. | x | x |
+| **6.4** | MSTG-PLATFORM-4 | La aplicación no debe exponer ninguna funcionalidad sensible a través de mecanismos IPC salvo que dichos mecanismos estén debidamente protegidos. | x | x |
+| **6.5** | MSTG-PLATFORM-5 | JavaScript debe estar deshabilitado en los WebViews salvo que sea necesario. | x | x |
+| **6.6** | MSTG-PLATFORM-6 | Las WebViews se deben configurar para permitir el mínimo número de protocolos requeridos (idealmente sólo https). Aquellos considerados como peligrosos deben estar deshabilitados (file, tel y app-id). | x | x |
```
Comment (cpholguera) [inline Document-es/0x11-V6-Interaction_with_the_environment.md:16]:
```suggestion
| **6.6** | MSTG-PLATFORM-6 | Las WebViews se deben configurar para permitir el mínimo número de protocolos requeridos (idealmente sólo https). Aquellos considerados como peligrosos deben estar deshabilitados (p. ej. file, tel y app-id). | x | x |
```

File: Document-es/0x12-V7-Code_quality_and_build_setting_requirements.md [hunk 15]
```diff
@@ -1,22 +1,22 @@
...
-| **7.9** | MSTG-CODE-9 | Las funcionalidades de seguridad gratuitas de las herramientas, tales como minificación del byte-code, protección de la pila, soporte PIE y conteo automático de referencias, se encuentran activadas. | x | x |
+| **7.1** | MSTG-CODE-1 | La aplicación debe estar firmada y provista con un certificado válido, cuya clave privada está debidamente protegida. | x | x |
+| **7.2** | MSTG-CODE-2 | La aplicación debe estar publicada en modo release y con sus configuraciones apropiadas (p. ej. non-debuggable). | x | x |
+| **7.3** | MSTG-CODE-3 | Los símbolos de depuración deben estar eliminados de los binarios nativos. | x | x |
+| **7.4** | MSTG-CODE-4 | Cualquier código de depuración y/o de asistencia al desarrollador (p. ej. código de test, backdoors, configuraciones ocultas) debe ser eliminado. La aplicación no debe hacer logs detallados de errores ni de mensajes de depuración. | x | x |
+| **7.5** | MSTG-CODE-5 | Todos los componentes de terceros (p. ej. librerías y frameworks) se encuentran identificados y sus vulnerabilidades conocidas revisadas. | x | x |
```
Comment (cpholguera) [inline Document-es/0x12-V7-Code_quality_and_build_setting_requirements.md:15]:
```suggestion
| **7.5** | MSTG-CODE-5 | Todos los componentes de terceros deben ser identificados y revisados en busca de vulnerabilidades comunes. | x | x |
```

File: Document-es/0x12-V7-Code_quality_and_build_setting_requirements.md [hunk 16]
```diff
@@ -1,22 +1,22 @@
...
+| **7.6** | MSTG-CODE-6 | La aplicación debe capturar y gestionar debidamente las posibles excepciones. | x | x |
+| **7.7** | MSTG-CODE-7 | Un correcto control de errores debe denegar el acceso por defecto. | x | x |
+| **7.8** | MSTG-CODE-8 | En el código no administrado, la memoria solicitada debe ser utilizada y liberada de manera correcta. | x | x |
+| **7.9** | MSTG-CODE-9 | Las funcionalidades y las herramientas de seguridad gratuitas deben estar activadas para: La simplificación del código, protección de la pila, soporte PIE y conteo automático de referencias. | x | x |
```
Comment (cpholguera) [inline Document-es/0x12-V7-Code_quality_and_build_setting_requirements.md:19]:
```suggestion
| **7.9** | MSTG-CODE-9 | Las funcionalidades y las herramientas de seguridad gratuitas deben estar activadas. Esto incluye simplificación del código, protección de la pila, soporte PIE y conteo automático de referencias. | x | x |
```

**Review by cpholguera — CHANGES_REQUESTED** [review]
Hi @antoniojturel, thank you very much for taking the time to do this corrections. I think it's a very nice choice to use "requisitos" and you've made the controls sound more native like, which is not always easy.

Please take a look at the suggestions. If you agree with them you can either click on "Commit suggestion" one by one or "Add suggestion to batch" and then commit once at the end.

Thank you!

File: Document-es/0x07-V2-Data_Storage_and_Privacy_requirements.md [hunk 17]
```diff
@@ -1,40 +1,40 @@
...
 
 ### Definición de Datos Sensibles
 
-Los datos sensibles en el contexto del MASVS se refieren tanto a las credenciales de usuario como a cualquier otro dato que se considere sensible en el contexto particular, por ejemplo:
+Los datos sensibles en el contexto del MASVS se refieren tanto a las credenciales de usuario como a cualquier otro dato que se considere sensible en el contexto particular de los siguientes ejemplos:
 
-- Información de identificación personal que puede ser usada para el robo de identidad: números de seguro social, números de tarjetas de crédito, números de cuentas bancarias, información médica;
-- Datos altamente confidenciales que, en caso de que se comprometieran, ocasionarían daños a la reputación y/o costes financieros: información contractual, información cubierta por acuerdos de confidencialidad, información de gestión;
+- Información de identificación personal que puede ser usada para el robo de identidad: Información de la Seguridad Social, números de tarjetas de crédito, números de cuentas bancarias, información médica.
```
Comment (cpholguera) [inline Document-es/0x07-V2-Data_Storage_and_Privacy_requirements.md:13]:
```suggestion
- Información de identificación personal que puede ser usada para el robo de identidad: información de la Seguridad Social, números de tarjetas de crédito, números de cuentas bancarias, información médica.
```

**Review by cpholguera — APPROVED** [review]
Hi @antoniojturel, I had to restore a couple of resolved items since we really need them in. Many of them were about the use of ":" which require lowercase after it in the cases we're dealing with (see [here](https://www.rae.es/dpd/dos%20puntos) for more info).

Regarding the rest of the changes I really like this new version, thank you very much!

Thanks a lot

**Comment by antoniojturel** [comment]
This PR is a full in depth review of 13 files, It wasn´t easy but effective step by step, great job @cpholguera !

**Pull request merged**